config/initializers/secure_headers.rb

```diff
@@ -1,12 +1,15 @@
 ::SecureHeaders::Configuration.default do |config|
-  config.hsts = false
+  config.hsts = "max-age=#{20.years.to_i}; includeSubdomains"
   config.x_frame_options = 'DENY'
   config.x_content_type_options = 'nosniff'
   config.x_xss_protection = '1; mode=block'
 
-  config.csp = false
+  config.csp = {
```

Comment:
Are you sure this is the same as before?

Reply:
@duduribeiro what about this?

Reply:
I saw in the doc that this is the default. But I will test if false is the same as this

Reply:
disabled it by default